guides/Developing Expo Go.md

```diff
@@ -1,94 +1,3 @@
-- Run `yarn` in the root directory.
-- Run `yarn setup:native` in the root directory.
-- Run `yarn build` in the `packages/expo` directory.
-
-### iOS
-
-- Make sure you have latest non-beta Xcode installed.
-- Run `et ios-generate-dynamic-macros`.
-- Open and run `ios/Exponent.xcworkspace` in Xcode.
-
-### Android
-
-- Make sure you have Android Studio 3 or newer installed.
-- See ["Running on a Device"](#running-on-a-device).
-
-## Running on a Device
-
-### iOS
-
-- In Xcode's menu bar, open the **Xcode** drop-down menu, and select **Preferences**. Then in the **Accounts** tab of the preferences menu, add your personal or team Apple Developer account.
-- Connect your test device to your computer with a USB cable.
-- In Xcode's menu bar, open the **Product** drop-down menu, select **Destination**, then in the _Device_ grouping select your device.
-- In the project navigator, select the **Exponent** project to bring up the project's settings, and then:
-  - In the **General** tab, in the **Identity** section, put in a unique Bundle Identifier.
-  - Also in the **General** tab, in the **Signing** section, select your personal or team Apple Developer account as your **Team**, and create a new signing certificate by clicking **Fix Issue**.
-- In the project's settings page, select the **ExpoNotificationServiceExtension** target, and then:
-  - In the **General** tab, in the **Identity** section, change its Bundle Identifier to be prefixed with your unique Bundle Identifier. For example, if your unique Bundle Identifier is `host.exp.Exponent.unique`, then the Bundle Identifier for the notification service extension should be `host.exp.Exponent.unique.ExpoNotificationServiceExtension`.
-- Finally, run the build.
-
-### Android
-
-- If the Play Store version of the Expo Go is installed on your test device, uninstall it.
-- Connect your test device to your computer with a USB cable.
-- Run `fastlane android start`, or alternately open the `android` directory in Android Studio, start it, and in the **Select Deployment Target** dialog, select your device.
-  - Expo Go can be built with either `versioned` or `unversioned` flavor. The latter is highly recommended for development and is used by default in Android Studio. You can switch flavors in _Build Variants_ pane.
-  - You can also run `./gradlew installUnversionedDebug` from the `android` directory.
-  - If you're having trouble building the Android app, trying clearing your gradle cache with `./gradlew clean` and rebuilding.
-
-## Modifying JS Code
-
-The Expo Go apps run a root Expo project in addition to native code. By default, this will use a published version of the project, so any changes made in the `home` directory will not show up without some extra work.
-
-Serve this project locally by running `npx expo start --port=80` from the `home` directory. **On iOS**, you'll additionally need to set `DEV_KERNEL_SOURCE` to `LOCAL` in `EXBuildConstants.plist` (the default is `PUBLISHED`).
```

Comment:
the `npx expo start --port=80` is quite important, please make sure to have it on the new doc.

Reply:
That's still included in step 2 - starting metro in expo go folder is configured to start on port 80 🙂.

Reply:
added "Metro needs to run prior running the build. Verify it runs on port 80. This is because `et android-generate-dynamic-macros` / `et ios-generate-dynamic-macros` is run during the build and needs Metro on port 80 to be running."